tekton/resources/images/bindings.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: TriggerBinding
+metadata:
+  name: trigger-to-build-and-push-image
+spec:
+  params:
+  - name: buildUUID
+    value: $(body.buildUUID)
+  - name: gitRepository
+    value: $(body.gitRepository)
+  - name: gitRevision
+    value: $(body.gitRevision)
+  - name: contextPath
+    value: $(body.contextPath)
+  - name: registry
+    value: $(body.registry)
+  - name: namespace
+    value: $(body.namespace)
+  - name: imageName
+    value: $(body.imageName)
+  - name: imageTag
+    value: $(body.imageTag)
+---
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: TriggerBinding
+metadata:
+  name: trigger-to-build-and-push-image-multi-arch
+spec:
+  params:
+  - name: buildUUID
+    value: $(body.buildUUID)
+  - name: gitRepository
+    value: $(body.gitRepository)
+  - name: gitRevision
+    value: $(body.gitRevision)
+  - name: contextPath
+    value: $(body.contextPath)
+  - name: registry
+    value: $(body.registry)
+  - name: namespace
+    value: $(body.namespace)
+  - name: imageName
+    value: $(body.imageName)
+  - name: imageTag
+    value: $(body.imageTag)
```

Comment:
Could you keep only `platforms` in this binding, and add both bindings in the event listener for the multi-arch case?